### PR TITLE
Update to MicroProfile 3

### PR DESCRIPTION
**Pull request by aguibert**
Includes:
- Update to microProfile 3
- Fix breaking changes between MP Metrics 2.2 --> 3.0
- Update to Jakarta EE 8 dependencies
- Update to OpenLiberty 19.0.0.9
- Use `baseUri` annotation attribute instead of JVM option for MP REST Clients

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -66,12 +66,12 @@ subprojects {
...
-        providedCompile group: 'org.eclipse.microprofile', name: 'microprofile', version: '2.2'
-        providedCompile group: 'javax', name: 'javaee-api', version: '8.0'
+        providedCompile group: 'org.eclipse.microprofile', name: 'microprofile', version: '3.0'
+        providedCompile group: 'jakarta.platform', name: 'jakarta.jakartaee-api', version: '8.0.0'
         testCompile group: 'junit', name: 'junit', version: '4.12'
-        testCompile group: 'org.eclipse', name: 'yasson', version: '1.0.3'
-        testCompile group: 'org.glassfish', name: 'jakarta.json', version: '1.1.5'
-        libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[19.0.0.8,)'
+        testCompile group: 'org.eclipse', name: 'yasson', version: '1.0.5'
+        //testCompile group: 'org.glassfish', name: 'jakarta.json', version: '1.1.5'
```
Comment (realModusOperandi) [inline build.gradle]: Accidental comment?

File: game-service/src/main/java/org/libertybikes/game/metric/GameMetrics.java [hunk 2]
```diff
@@ -83,13 +95,15 @@ private static MetricRegistry getRegistry() {
 
     public static void counterInc(Metadata metricMetadata) {
         if (registry != null || (getRegistry() != null)) {
-            registry.counter(metricMetadata).inc();
+            registry.concurrentGauge(metricMetadata).inc();
+            //registry.counter(metricMetadata).inc();
```
Comment (realModusOperandi) [inline game-service/src/main/java/org/libertybikes/game/metric/GameMetrics.java]: Accidental comment?

File: game-service/src/main/java/org/libertybikes/game/metric/GameMetrics.java [hunk 3]
```diff
@@ -83,13 +95,15 @@ private static MetricRegistry getRegistry() {
...
         }
     }
 
     public static void counterDec(Metadata metricMetadata) {
         if (registry != null || (getRegistry() != null)) {
-            registry.counter(metricMetadata).dec();
+            registry.concurrentGauge(metricMetadata).dec();
+            //registry.counter(metricMetadata).dec();
```
Comment (realModusOperandi) [inline game-service/src/main/java/org/libertybikes/game/metric/GameMetrics.java]: Accidental comment?

File: build.gradle (hunk 1 above)
Comment (aguibert) [reply build.gradle]: yea, I commented it out while I was testing its removal. I'll remove this commented out line and the other 2 occurrences in my DB PR that builds on top of this branch

**Pull request merged**